x/fungible/keeper/evm.go

```diff
@@ -283,7 +283,7 @@ func (k Keeper) UpdateZRC20ProtocolFlatFee(
 		types.ModuleAddressEVM,
 		zrc20Addr,
 		BigIntZero,
-		nil,
+		big.NewInt(100_000),
```

Comment:
Can we maybe use 200k?

I don't think there would be any deifferent and might save some hassles in somehow you get a out of gas with 100k